Spoj_problems/DIVFACT/Solution.cpp

```diff
@@ -0,0 +1,71 @@
+#include<bits/stdc++.h>
+using namespace std;
+#define ll long long int
+#define m 1000000007
+//FIND PRIMES IN [1,N]  USING EXTRA SPACE WITH TIME COMPLEXITY O(N LOG (LOG (N)))
+vector<int > v;
+void seive(ll n)
+{
+    ll c=0;
+    bool a[n+1];
+    for(ll i=0;i<=n+1;i++){
+    a[i]=true;
+    }
+    a[0]=a[1]=false;
+    for(ll i=2;i*i<=n;i++)
+    {
+        if(a[i]==true)
+        {
+            for(ll j=i*i;j<=n;j+=i)// MAKE FALSE ALL THE MULTIPLES OF NUMBERS WHOSE POSITION IS INITIALLY TRUES
+            {
+                a[j]=false;
+            }
+        }
+    }
+    for(ll i=2;i<=n;i++)
+    {
+        if(a[i]==true)
+        {
+            v.push_back(i);
+        }
+    }
+   /* for(ll i=0;i<v.size();i++)
+    cout<<v[i]<<" ";
```

Comment:
Please remove such unnecessary commented codes. Or if it is something needed, mention it's purpose
Also please add some more comments briefly explaining your solution.